evals/gaia/gaia.py

```diff
@@ -0,0 +1,93 @@
+from huggingface_hub import snapshot_download
+from pathlib import Path
+from inspect_ai import task
+from inspect_ai.scorer import Score, match
+from inspect_ai.solver import basic_agent, system_message, TaskState
+from inspect_ai.tool import web_search, bash
+from inspect_ai.scorer import scorer, Target
+from inspect_ai.dataset import Sample, hf_dataset
+from typing import Callable, Literal
+from inspect_ai import Task
+from inspect_ai.dataset._dataset import FieldSpec
+import os
+from typing import Sequence
+
+# TODO: The constants actaully make thins a bit uglier, I'd love it if people were like "wow look at how clean it is to onboard the dataset"
+DEFAULT_INPUT_PROMPT = """Please answer the question below. You should:
+
+- Return only your answer, which should be a number, or a short phrase with as few words as possible, or a comma separated list of numbers and/or strings.
+- If the answer is a number, return only the number without any units unless specified otherwise.
+- If the answer is a string, don't include articles, and don't use abbreviations (e.g. for states).
+- If the answer is a comma separated list, apply the above rules to each element in the list.
+
+Any files or attachments mentioned in the question can be found in the /shared_files/ directory (some questions do not have associated files). Here is the question:
+
+{question}"""
+
+# TODO: This is a little awkward, since initalizing the web search tool requires you to have an API key, but I don't want to force people to have to download that
+DEFAULT_PLAN = basic_agent(
+    init=system_message(
+        "Please solve the coding task below. Once you are done, use your submit tool."
+    ),
+    tools=[bash()],  # + [web_search()]
+)
```

Comment:
Some awkwardness here, where the web_search() tool asks for a google API key on initalisation, but I don't want to force people to have on to import the module.

Reply:
The other thing to do is have `default_plan()` be a function rather than a constant.

Reply:
Done!